pkg/streamingpromql/operators/aggregation.go

```diff
@@ -23,17 +23,38 @@ import (
 )
 
 type Aggregation struct {
-	Inner    types.InstantVectorOperator
-	Start    time.Time
-	End      time.Time
-	Interval time.Duration
-	Grouping []string
-	Pool     *pooling.LimitingPool
+	inner    types.InstantVectorOperator
+	start    int64
+	end      int64
+	interval int64
```

Comment:
[nit] Might be good to add a comment explaining the units of these (`Selector` has the same fields, you could steal the comments there)